<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberDlg</class>
 <widget class="QDialog" name="MemberDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Member information</string>
  </property>
  <layout class="QVBoxLayout" name="dlgVerticalLayout">
   <item>
    <layout class="QHBoxLayout" name="baseHorizontalLayout">
     <item>
      <widget class="QGroupBox" name="avatarGroupBox">
       <property name="title">
        <string>Avatar</string>
       </property>
       <layout class="QVBoxLayout" name="avatarVerticalLayout">
        <item>
         <widget class="QLabel" name="avatarLabel">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>350</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>350</height>
           </size>
          </property>
          <property name="pixmap">
           <pixmap resource="../resouce.qrc">:/images/avatar.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="avatarHorizontalLayout">
          <item>
           <widget class="QPushButton" name="uploadButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="downloadButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Download</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="infoGroupBox">
       <property name="title">
        <string>Information</string>
       </property>
       <layout class="QVBoxLayout" name="infoVerticalLayout">
        <item>
         <widget class="QTabWidget" name="infoTab"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="listGroupBox">
     <property name="title">
      <string>List</string>
     </property>
     <layout class="QHBoxLayout" name="listHorizontalLayout">
      <item>
       <widget class="QTabWidget" name="listTab"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="listVerticalLayout">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="btnHorizontalLayout">
     <item>
      <spacer name="btnHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uploadButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resouce.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MemberDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MemberDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
